CHANGELOG.md

```diff
@@ -7,28 +7,54 @@ vNext
 (Add your change to a random empty line to avoid merge conflicts)
  -
  -
- - Added an NLP text classification example (SST-2 sentiment) to examples/sst2.
-   that uses a birectional LSTM (BiLSTM) to encode the input text.
- - Added flax.training.train_state to simplifying using Optax optimizers.
- - Rewrote ImageNet example to use Optax instead of flax.optim for optimizers.
  -
- - `mutable` argument is now available on `Module.init` and `Module.init_with_outputs`
- - When calling `init` the 'intermediates' collection is no longer mutable
-   Therefore, intermediates will no longer be returned from initialization by default. 
  -
  -
- - Bug Fix: Correclty handle non-default parameters of Linen Modules with nested inheritance.
  -
  -
  -
  -
- - `BatchNorm` instances will behave correctly during init when called multiple times.
  -
  -
  -
  -
  -
+ -
+ -
+ -
+ -
+ -
+ -
+ -
+ -
+ -
+
+0.3.4
+------
+
+Possibly breaking changes:
+ - When calling `init` the 'intermediates' collection is no longer mutable.
+   Therefore, intermediates will no longer be returned from initialization by default. 
+ - Don't update batch statistics during initialization.
+ - Attention: require deterministic only if using dropout
+
 
+Other changes:
+ - Rewrote various examples to use Optax instead of Flax optimizers (e.g., Imagenet, SST2)
+ - Added an NLP text classification example (SST-2 sentiment) to examples/sst2.
+   that uses a birectional LSTM (BiLSTM) to encode the input text.
```

Comment:
Suggestions to improve this part:

- Fix a typo: "bidirectional" (can't keep up with all the papers, so I'm assuming you are referring to this BiLSTM: _Bidirectional Recurrent Neural Networks_ (Schuster & Paliwal, 1997) https://maxwell.ict.griffith.edu.au/spl/publications/papers/ieeesp97_schuster.pdf).
- Add a link to the SST-2 text classifier example to help users find it quicker.

```suggestion
 - Added an NLP text classification example (on the SST-2 dataset) to
   [`examples/sst2`](https://github.com/google/flax/tree/master/examples/sst2).
   that uses a bidirectional LSTM (BiLSTM) to encode the input text.
```